src/components/EmbeddedWalletProfileDropdown/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { Avatar, Button, Icon } from "@stellar/design-system";
+
+import { DropdownMenu } from "@/components/DropdownMenu";
+import { EmbeddedWalletReceiverContact } from "@/types";
+
+import "./styles.scss";
+
+type Props = {
+  contact: EmbeddedWalletReceiverContact;
+  onOpenProfile: () => void;
+  onLogout: () => void;
+};
+
+const CONTACT_ICON_MAP: Record<string, typeof Icon.UserCircle> = {
```

Comment:
I think the `React.ReactElement` for icon type should work.

```suggestion
const CONTACT_ICON_MAP: Record<string, React.ReactElement> = {
```